docs/Reference/MPS/configuration.md

```diff
@@ -1,25 +1,63 @@
 --8<-- "References/abbreviations.md"
-# MPS Configuration Options
+## MPS `.env` Configuration
```

Comment:
These shouldn't be separate tables, Env variables overwrite their .rc counterparts, if anything it should be another column in the table. But a separate table entirely make it seem like they are different sets of options when they are not.